test/classification.jl

```diff
@@ -44,8 +44,16 @@ random_second_input(op) = random_input(op)
 ## 1-to-1
 
 function correct_classification_1_to_1(op, x; atol)
-    dfdx = derivative(op, x)
-    d²fdx² = second_derivative(op, x)
+    dfdx, d²fdx² = try
+        derivative(op, x), second_derivative(op, x)
+    catch e
+        if e isa MethodError
+            # ForwardDiff can't handle this operator: assume it has zero derivatives
+            0.0, 0.0
```

Comment:
Why not manually add dispatches on the operator types for zero-derivative operators as we previously did? 
I feel like this would be a bit safer than simply silencing `MethodError`s.

Reply:
done in the last commit